Server/Static/management/commands/db_init.py

```diff
@@ -0,0 +1,29 @@
+from django.core.management.base import BaseCommand
+from Static.create_data import *
+from Static.models import Version
+
+class Command(BaseCommand):
+
+	help = "This file will initialize the database using the inputted <version> data"
+
+	def add_arguments(self, parser):
+		parser.add_argument('version', nargs="+", type=str)
+
+	def handle(self, *args, **options):
+		for version in options['version']:
+			try:
+				ver = Version.objects.get(name=version)
+			except Exception:
+				pass
+			else:
+				self.stderr.write("Version already exists in database, exiting...")
```

Comment:
It does.  The exception means the version does not yet exist.  If it's triggered, then the else is not.